skfp/bases/base_filter.py

```diff
@@ -6,11 +6,11 @@
 
 import numpy as np
 from joblib import effective_n_jobs
-from rdkit.Chem import Mol
+from rdkit.Chem.rdchem import Mol
```

Comment:
Unnecessary, we use `from rdkit.Chem import Mol` in all files

Reply:
Reverted in commit [f65c196](https://github.com/scikit-fingerprints/scikit-fingerprints/pull/298/commits/f65c196709b448510631846e9511c6f799b97789)

Reply:
The reason for the change was to remove pylint errors ranting about it. I agree it is one of those discutible errors.